source/hardware/controllers/numark_mixtrack_platinum_fx.rst

```diff
@@ -0,0 +1,866 @@
+.. Credit to PopHippy for creating the original PDF file.
+
+Numark Mixtrack Platinum FX
+
+==============================
+
+-  `Manufacturer’s product page <https://www.numark.com/product/mixtrack-platinum-fx>`__
+-  `Forum thread <https://mixxx.discourse.group/t/numark-mixtrack-platinum-fx-mapping/19985/215>`__
+
+Mixtrack Platinum FX is a 24-bit 4-deck digital DJ controller with a dedicated software effects section, large 6-inch capacitive-touch jog wheels with built in hi-res displays and 16-multifunctional performance pads.
+
+   Firmware
+
+   The mapping requires the following Soft/Firmware for all enabled
+   features to work for Mixxx 2.4
+
+-  `Firmware Update v1.10 MAC <https://cdn.inmusicbrands.com/Numark/MixtrackPlatinumFX/Mixtrack%20Platinum%20FX%20Firmware%201.10%20Mac.zip>`__
+-  `Firmware Update v1.10 WIN <https://cdn.inmusicbrands.com/Numark/MixtrackPlatinumFX/Mixtrack%20Platinum%20FX%20Firmware%201.10%20Win.zip>`__
+- Linux user need to install firmware using Windows or MAC.
+
+
+Overview
+--------
+
+.. figure:: ../../_static/controllers/numark_mixtrack_platinum_fx_toprow.svg
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: Numark Mixtrack Playinum FX (schematic view)
+   :figclass: pretty-figures
+
+---------------
```

Comment:
```suggestion
---------------
```
Please remove these lines. They prevent proper header rendering. Also some table rows are broken due to blank lines.
I suggest you check the preview and tweak until it looks as desired.
https://deploy-preview-623--mixxx-manual.netlify.app/hardware/controllers/numark_mixtrack_platinum_fx

Reply:
I removed the lines.

Reply:
The appearance is still broken, just check the preview.
FYI you may also build the html locally with `make html` and double-check before pushing.

Reply:
I just updated the appearance. Can you recheck and make suggestion on which areas need to be improve?

Reply:
Yes, it's loking good now. It were just the blank lines that enforced header formatting on each following line.